### ISSUE TITLE
Update the order of elements in the advanced filter dialog

### ISSUE THREAD

**Issue by jmatth** [jmatth/dextools#48]
They should be ordered in the most convenient way to navigate via the keyboard, then positioned more conveniently via the various order props in the vuetify grid system.
